src/cities/cityConfigs.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+import CityConfig from './CityConfig'
+
+const configs = [
+  new CityConfig('neuburgschrobenhausenwohnraum', 'neuburg.wohnen.integreat-app.de', true, 'Neuburg-Schrobenhausen', null, 'https://www.neuburg-schrobenhausen.de/Impressum.n10.html'),
+  new CityConfig('bayreuthwohnraum', 'bayreuth.wohnen.integreat-app.de', false, 'Bayreuth', null, 'https://www.bayreuth.de/impressum/')
```

Comment:
What is null?

Reply:
I think we already have a logo for neuburg

Reply:
I know, but since we don't need it in the backend and also don't offer an api yet I didn't add it.